src/python/bot/fuzzers/syzkaller/runner.py

```diff
@@ -58,22 +58,21 @@ def get_config():
 
 def get_runner(fuzzer_path):
   """Return a syzkaller runner object."""
-  build_dir = environment.get_value('BUILD_DIR')
-  return AndroidSyzkallerRunner(fuzzer_path, build_dir)
+  return AndroidSyzkallerRunner(fuzzer_path)
 
 
-class AndroidSyzkallerRunner(new_process.ProcessRunner):
+class AndroidSyzkallerRunner(new_process.UnicodeProcessRunner):
   """Syzkaller runner."""
 
-  def __init__(self, executable_path, default_args=None):
+  def __init__(self, executable_path):
     """Inits the AndroidSyzkallerRunner.
 
     Args:
       executable_path: Path to the fuzzer executable.
       default_args: Default arguments to always pass to the fuzzer.
     """
-    super(AndroidSyzkallerRunner, self).__init__(
-        executable_path=executable_path, default_args=default_args)
```

Comment:
Previously default_args=None, this was regressed and changed to default_args incorrectly which led to this:
```
syz-manager / u s r / l o c a l / g o o g l e / h o m e / a n d r o i d - t e s t / c l u s t e r f u z z / 8 0 1 K P Y R 1 3 6 0 0 7 4 / b o t / c l u s t e r f u z z / b o t / b u i l d s / s y z k a l l e r _ k a s a n _ a n d r o i d _ d e v i c e / c u s t o m -config 
``` :)

Reply:
What passed this? Seems like an issue with the caller rather than this being the problem.

Reply:
yes caller is get_runner which is fixed as well in this CL.